packages/server/graphql/mutations/dragEstimatingTask.ts

```diff
@@ -48,16 +45,22 @@ export default {
       return standardError(new Error('Meeting phase not found'), {userId: viewerId})
     }
     const {stages} = estimatePhase
-    const draggedStage = stages.find((stage) => stage.id === stageId)
-    if (!draggedStage) {
-      return standardError(new Error('Meeting stage not found'), {userId: viewerId})
+    const draggedStages = stages.filter((stage) => stageIds.includes(stage.id))
+    if (!draggedStages.length) {
+      return standardError(new Error('No meeting stages were found'), {userId: viewerId})
     }
 
     // RESOLUTION
     // MUTATIVE
-    draggedStage.sortOrder = sortOrder
+    draggedStages.forEach((draggedStage) => (draggedStage.sortOrder = sortOrder))
```

Comment:
can use same logic as in the client. the noise will be different, but the order will be the same